src/components/main-view/main-view.jsx

```diff
@@ -2,63 +2,117 @@ import React, { useState, useEffect } from 'react';
 import { MovieCard } from "../movie-card/movie-card";
 import { MovieView } from "../movie-view/movie-view";
 import { LoginPage } from "../login-view/login-page";
-import "./main-view.css"
+import { SignUp } from '../signup-view/signup-view';
+import "./main-view.css";
+import { Button, Row, Col } from "react-bootstrap";
 
 export const MainView = () => {
-  const [movies, setMovies] = useState([]);
 
+  const storedToken = localStorage.getItem("token");
+  const storedUser = JSON.parse(localStorage.getItem("user"));
+  const [movie, setMovie] = useState([]);
```

Comment:
Here I would name it `movies` in plural because it's an array.

Reply:
+